streaming/stream_session.cc

```diff
@@ -220,14 +223,27 @@ void stream_manager::init_messaging_service_handler(abort_source& as) {
                     if (try_catch<seastar::rpc::stream_closed>(ex)) {
                         level = seastar::log_level::debug;
                     }
+                    bool aborted = false;
+                    if (auto nested = try_catch<nested_exception>(ex)) {
+                        if (try_catch<rpc::canceled_error>(nested->inner)) {
+                            level = seastar::log_level::info;
+                            aborted = true;
+                        }
+                    }
                     status = -1;
                     // The status code -2 means error and the table is dropped
                     if (db.local_is_initialized() && mm.local_is_initialized() && co_await table_sync_and_check(db.local(), mm.local(), cf_id)) {
                         level = seastar::log_level::debug;
                         status = -2;
                     }
-                    sslog.log(level, "[Stream #{}] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks={}, cf={}, peer={}: {}",
-                            plan_id, s->ks_name(), s->cf_name(), from, ex);
+                    if (aborted) {
+                        sslog.log(level, "[Stream #{}] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks={}, cf={}, peer={}: Streaming aborted",
+                                plan_id, s->ks_name(), s->cf_name(), from);
+                    }
+                    else {
+                        sslog.log(level, "[Stream #{}] Failed to handle STREAM_MUTATION_FRAGMENTS (receive and distribute phase) for ks={}, cf={}, peer={}: {}",
+                                plan_id, s->ks_name(), s->cf_name(), from, ex);
```

Comment:
I think these two could be united, by feeding the log level and extra message to a single call to `sslog.log`.